src/components/portrait/portrait-initials.spec.js

```diff
@@ -54,7 +53,7 @@ describe('PortraitInitials', () => {
     });
 
     it('clears the cached initials if theme changes', () => {
-      props.theme = smallTheme;
+      props.theme = carbonThemeList[1];
```

Comment:
I'm not sure about this approach of accessing a specific numeric index of `carbonThemeList`. It feels like a [magic number](https://en.wikipedia.org/wiki/Magic_number_%28programming%29#Unnamed_numerical_constants) situation, which _"obscures the developers' intent in choosing that number"_. In my opinion, it'd be clearer to import and use the specific themes individually:
```js
import mintTheme from '../../style/themes/mint';
import aegeanTheme from '../../style/themes/aegean';
```
```jsx
<ThemeProvider theme={ mintTheme }>
```
```js
props.theme = aegeanTheme;
```

Reply:
I have added two constants to make it clearer why these themes are picked from the list. What do you think about that approach?

Reply:
I think that's an improvement, thanks :thumbsup: https://github.com/Sage/carbon/commit/8e55c3204b58536aec3fc01bbea09a3be48cff0c#diff-9f8288a23e2b9582ebfd397257c87a8d